docs/semgrep-app/pricing-and-billing.mdx

```diff
@@ -1,53 +1,167 @@
+
+### Plan limits
+
+Should your plan exceed the limit, we will reach out and come up with a new plan that fits your needs.
 
-Yes, you can use the Semgrep CLI and Semgrep Community Tier for [free](https://semgrep.dev/getting-started)!
+## Upgrading your plan
 
-## What are the differences between the free and paid plans?
+To upgrade to the **Team tier** through **credit card**:
 
-Semgrep Community is great for individuals looking to quickly secure their public and private projects and teams looking to explore and try out the product.
+1. In the Settings page, select the Payment tab.
+2. Select the number of seats to purchase.
+3. Fill in your payment details.
 
-Semgrep Team and Enterprise are best for larger organizations that require dedicated support and advanced functionality.
 
-You can find more on the [pricing page](https://semgrep.dev/pricing).
+![Screenshot of Semgrep App payment menu](../img/billing-and-pricing-payment.png)<br />
 
-## How many seats do I need?
+To upgrade to the **Enterprise tier**, contact us through our [Semgrep for organizations](https://semgrep.dev/contact) page.
 
-One seat corresponds to one project contributor whose merge/pull request or code is scanned during the billing period.
+### Billing
 
-For example, if a project has 4 unique contributors who push code during the billing period and Semgrep runs on their merge/pull requests, 4 seats are required. If these unique contributors commit to many projects within the same organization, they will only be counted once, so no additional seats are required.
+Team tier users are charged monthly. Enterprise tier users are charged at an agreed-upon billing cycle. For any concerns, send an email to [billing@r2c.dev](mailto:billing@r2c.dev) to get in touch with our sales team.
 
-## How do I upgrade my plan?
+## Modifying or canceling your plan
 
-To upgrade to the Team tier, select the number of seats you wish to purchase and pay directly from the Settings page in Semgrep App!
+To modify or cancel your plan, send an email to [billing@r2c.dev](mailto:billing@r2c.dev).
 
-For Enterprise tier pricing, please contact us via the form on the [pricing page](http://semgrep.dev/pricing).
+## Paying for your plan
 
-## How do I pay for my plan?
+Pay through the following methods: 
 
-We accept all major credit cards through [Stripe](https://stripe.com/). If you want to pay through a purchase order / invoice mechanism, you can also reach out to [billing@r2c.dev](mailto:billing@r2c.dev) to get in touch with our sales team.
+<dl>
+<dt>Pay using your credit card.</dt>
+<dd>The payment will be processed through Stripe.</dd>
+<dt>Pay through a purchase order or invoice.</dt>
+<dd>Send an email to [billing@r2c.dev](mailto:billing@r2c.dev) to get in touch with our sales team.</dd>
```

Comment:
For some reason it seems this email link isn’t rendering properly. I have stared at it and can’t figure out what’s different here vs. other places that email is used similarly.

Reply:
It was my mistake for not checking. Mixing tags and markdown syntax in the same line is not recommended. Corrected by using anchor tags.